src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs

```diff
@@ -35,18 +38,57 @@ namespace Windows.UI.Xaml.Controls
 {
 	public partial class ScrollViewer : ContentControl, IFrameworkTemplatePoolAware
 	{
-		internal const string ScrollContentPresenterPartName = "ScrollContentPresenter";
+		private static class Parts
+		{
+			public static class Uwp
+			{
+				public const string ScrollContentPresenter = "ScrollContentPresenter";
+				public const string VerticalScrollBar = "VerticalScrollBar";
+				public const string HorizontalScrollBar = "HorizontalScrollBar";
+			}
+
+			public static class WinUI3
+			{
+				public const string Scroller = "PART_Scroller";
+				public const string VerticalScrollBar = "PART_VerticalScrollBar";
+				public const string HorizontalScrollBar = "PART_HorizontalScrollBar";
+			}
+		}
+
+		private static class VisualStates
+		{
+			public static class ScrollingIndicator
+			{
+				public const string None = "NoIndicator";
+				public const string Touch = "TouchIndicator";
+				public const string Mouse = "MouseIndicator";
+				// public const string MouseFull = "MouseIndicatorFull"; // No supported yet
+			}
+
+			public static class ScrollBarsSeparator
+			{
+				public const string Collapsed = "ScrollBarSeparatorCollapsed";
+				public const string Expanded = "ScrollBarSeparatorExpanded";
+				public const string ExpandedWithoutAnimation = "ScrollBarSeparatorExpandedWithoutAnimation";
+				public const string CollapsedWithoutAnimation = "ScrollBarSeparatorCollapsedWithoutAnimation";
+
+				// On WinUI3 visuals states are prefixed with "ScrolBar***s***" (with a trailing 's')
+				//public const string Collapsed = "ScrollBarsSeparatorCollapsed";
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5975891)